pmd-core/src/main/java/net/sourceforge/pmd/rules/RuleBuilder.java

```diff
@@ -47,6 +47,14 @@
     private boolean isUsesMultifile;
     private boolean isUsesTyperesolution;
 
+    @Deprecated
+    public RuleBuilder(String name, String clazz, String language) {
+        this.name = name;
```

Comment:
Please call here just the other constructor:

```
this(name, new ResourceLoader(), clazz, language);
```

No need to duplicated the code here.

Reply:
Resolved